dev-environment/nussknacker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:buster
```

Comment:
In `docker-compose.yml` we have `postgres` from Alpine Linux and here it's a full Debian. Can we use Alpine for now? It will be a smaller image.

Reply:
alpine has strange compiler, packages dependencies, etc. so i ccan try change it to debian buster slim at most I think(alpine s not compatible with what we have on our workstaions). But when it is used for psql it's good because it only exposes database port to allow us data saving

Reply:
Additionally I think for nussknacker we should use something similar o our systems